client/state/data-getters/index.js

```diff
@@ -160,3 +160,29 @@ export const requestSiteAlerts = siteId => {
 		}
 	);
 };
+
+/**
+ * Request a site's frame nonce from the v1.3 sites endpoint.
+ *
+ * @param {number} siteId  Site Id.
+ * @return {*} Stored data container for request.
+ */
+export const requestFrameNonce = siteId => {
+	const id = `frame-nonce-${ siteId }`;
+
+	return requestHttpData(
+		id,
+		http(
+			{
+				apiVersion: '1.3',
+				method: 'GET',
+				path: `/sites/${ siteId }`,
```

Comment:
since we're discarding everything else we are free to send less data. for some reason the endpoint wasn't working the way I expected it to because it sent way more data than I asked for, but here is what we would add if it were working. note, that the `fields` property prunes out unwanted fields, but it didn't do so when the only field I requested was `options`, so I have included the `ID` as well - this is a workaround

```js
method: 'GET',
path: `/sites/${ siteId }`,
query: { fields: 'ID,options', options: 'frame_nonce' }
```

on my test site it 1.24 KB vs 2.71 KB so not much but something